### PR TITLE
Fix double error tags when option parsing fails.

### PR DESCRIPTION
**Pull request by markstory**
When err() started adding formatting for us, we forgot to remove the formatting added when handling option parser errors.

Refs #9394

### CODE REVIEW

**Comment by dereuromark** [comment]
👍

**Comment by dereuromark** [comment]
I do see 2 more of those:

- LoadTask::main()
- UnloadTask::main()

**Comment by markstory** [comment]
Thanks for pointing those out @dereuromark. I've removed the formatting on those as they'll get formatted by err().

File: src/Shell/Task/UnloadTask.php [hunk 1]
```diff
@@ -41,8 +41,8 @@ public function main($plugin = null)
         $this->bootstrap = ROOT . DIRECTORY_SEPARATOR . 'config' . DIRECTORY_SEPARATOR . 'bootstrap.php';
 
         if (empty($plugin)) {
-            $this->err('<error>You must provide a plugin name in CamelCase format.</error>');
-            $this->err('To unload an "Example" plugin, run <info>`cake plugin unload Example`</info>.');
+            $this->err('You must provide a plugin name in CamelCase format.');
+            $this->err('To unload an "Example" plugin, run <info>`cake plugin unload Example`.');
```
Comment (dereuromark) [inline src/Shell/Task/UnloadTask.php]: looks like an <info tag left here

**Comment by dereuromark** [comment]
PS: There seem to be quite a few mistakes in the Shell layer:

    public function warn(
        return $this->_io->err()

is returning int/bool on a void result.
That can't work.
Either way make those also void or wefix the ConsoleIO to not return void here and pass the info up.

**Comment by markstory** [comment]
@dereuromark Seems like we could fix the doc block data in a separate pull request.

**Pull request merged**